superset-frontend/packages/superset-ui-core/src/chart/models/ChartMetadata.ts

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { SqlaFormData } from '../../query';
```

Comment:
Let's use an absolute path to this. I believe we use absolute paths for whatever needs to go back of more than on directory.

Reply:
Changed to `QueryFormData` for consistency and imported from `@superset-ui/core` – does that sound good?